src/odemis/gui/model/tab_gui_data.py

```diff
@@ -488,7 +532,12 @@ def __init__(self, main):
 
         self.stage_align_slider_va = model.FloatVA(1e-6)
         self.show_advaned = model.BooleanVA(False)
+        self.view_posture = VigilantAttribute(FM_IMAGING)
+        # self.main.posture_manager.current_posture.subscribe(self._on_posture_change, init=True) # TODO: enable once new pm is merged
```

Comment:
what is "new pm"?

Reply:
https://github.com/delmic/odemis/pull/3005